cloud/credentials.go

```diff
@@ -38,6 +38,13 @@ type Credential struct {
 
 	// Label is optionally set to describe the credentials to a user.
 	Label string
+
+	// Invalid is true if the credential is invalid.
+	Invalid bool
```

Comment:
What's the difference between Revoked and Invalid?
Should we consider consolidating these? They sorta would be used to achieve the same thing right? Especially since we have InvalidReason. The idea was that a user would Revoke a credential with a reason for doing so.

Reply:
Revoked is used via JAAS. Invalid is used when cloud complains about a previously working credential. 
They are 2 separate concepts.